dao/src/main/java/org/thingsboard/server/dao/timeseries/CassandraBaseTimeseriesDao.java

```diff
@@ -85,6 +86,20 @@ public class CassandraBaseTimeseriesDao extends AbstractCassandraBaseTimeseriesD
     public static final String ASC_ORDER = "ASC";
     public static final long SECONDS_IN_DAY = TimeUnit.DAYS.toSeconds(1);
     protected static final List<Long> FIXED_PARTITION = List.of(0L);
+    private static final String INSERT_QUERY_WITH_NULL = INSERT_INTO + ModelConstants.TS_KV_CF +
```

Comment:
Please make a parametrized test that will save each data type. The test should fail if you swap columns, for example, swap here string and JSON. or long with TTL.